### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by ease-ln**

### CODE REVIEW

File: app_nodejs/app.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+`This module shows Moscow Time.`
+
+const express = require('express')
+const request = require('request');
+
+const PORT = 5000;
```
Comment (aldeeyar) [inline app_nodejs/app.js:6]: You might want to read port from environment variable

File: app_nodejs/app.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        if (response && response.statusCode == 200) {
+            res.send(JSON.parse(body).datetime.split('T')['1'].split('.')['0']);
+        } else {
+            res.send('No responce from worldtimeapi 404;)')
+        }
+      });      
+});
+
+app.listen(PORT, function (){ 
+    console.log('You web app on: http://localhost:5000/');
```
Comment (aldeeyar) [inline app_nodejs/app.js]:
```suggestion
    console.log(`Your web app on: http://localhost:${PORT}/`);
```

File: app_python/moscow_time/main.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+@app.route('/')
+def moscow_time():
+    url = "http://worldtimeapi.org/api/timezone/Europe/Moscow"
+    response = requests.get(url)
+    pattern = "%Y-%m-%dT%H:%M:%S.%f%z"
+    if response.status_code == 200:
+        date = datetime.strptime(response.json()['datetime'], pattern)
+        time = f"{date.hour}:{date.minute}:{date.second}"
+    return f"Moscow time: {time}"
```
Comment (aldeeyar) [inline app_python/moscow_time/main.py:18]: what if response status code is not 200? 
what value will have "time" variable

File: app_nodejs/app.js (hunk 1 above)
Comment (ease-ln) [reply app_nodejs/app.js:6]: thanks

**Pull request closed without merging**